# Problem Statement 1:
### Blood glucose levels for obese patients have a mean of 100 with a standard deviation of 15. A researcher thinks that a diet high in raw cornstarch will have a positive effect on blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an effect or not.

In [1]:
import pandas as pd
import numpy as np
import math as m
import scipy.stats
import matplotlib.pyplot as plt
from scipy.stats import binom

In [2]:
# H0=100
# H1!=100
# lets assume significance level as 2%
z=(108-100)/(15/36**0.5)
z

3.2

##### Statement:
##### from z table i got probability for 3.2 is 0.9993. i.e. probability of having value less than 108 is 0.9993 and more than or equals to 108 is (1-0.9993)=0.0007.
##### It is less than 0.02 so we will reject the Null hypothesis i.e. there is raw cornstarch effect.

# Problem Statement 2:
### In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple random sample of 100 voters are surveyed from each state. 
### What is the probability that the survey will show a greater percentage of Republican voters in the second state than in the first state?

In [3]:
# P1<P2

In [4]:
# mean of mean of the difference in sample proportions: republican voters
MR=(0.52-0.47)
MR

0.050000000000000044

In [5]:
# Standard Deviation of difference 
s=(((0.52)*(0.48)/100)+(0.47)*(0.53)/100)**0.5
s

0.07061869440877536

In [6]:
p1=0.52
p2=0.47
x=0
s=0.07061
z=(x-(p1-p2))/s
z

-0.7081149978756556

##### from Z scroe table probability for -0.70 is 0.24196 which is 24.19%
##### Therefore, the probability that the survey will show a greater percentage of Republican voters in the second state than in the first state is 0.24.

# Problem Statement 3:
### You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard deviation is 209. How well did you score on the test compared to the average test taker?

In [7]:
X=1100
M=1026
S=209
Z=(1100-1026)/209
Z

0.35406698564593303

##### from Zscore table probability for 0.35 is 0.6368 which is 63.68%
##### You scored 63.68% above in SAT compared to other average test takers.